src/backend/looped/weapons/no_crosshair.cpp

```diff
@@ -0,0 +1,12 @@
+#include "backend/looped/looped.hpp"
+#include "natives.hpp"
+
+namespace big
+{
+	void looped::weapons_no_crosshair()
+	{
+		if (g->weapons.no_crosshair) {
+			HUD::HIDE_HUD_COMPONENT_THIS_FRAME(14 /*RETICLE*/);
```

Comment:
Can't you already do that with #553?

https://github.com/YimMenu/YimMenu/blob/1087146e5670f6710a84cc841586a18f6db8055c/src/backend/looped/self/hud.cpp#L45-L47

Reply:
All I did was took the functionality of src/backend/looped/weapons/force_crosshair.cpp and copy pasted it to hide the component instead. I might switch to what you suggested though.

Reply:
Nah he's just saying that the functionality already exists thus it's no longer required.
So, you can just remove the code to hide the crosshair.
According to the code it should be possible to hide the RETICLE.

Reply:
I see what you mean now. I will remove that rn